covenant/guest-aggregation/src/main.rs

```diff
@@ -1,32 +1,39 @@
 #![no_main]
 zkm_zkvm::entrypoint!(main);
 
+use revm_primitives::B256;
 use sha2::{Digest, Sha256};
 
-pub fn main() {
-    // Read block numbers.
-    let block_numbers = zkm_zkvm::io::read::<Vec<u64>>();
+use crate::io::ZKMPublicValues;
+
+mod io;
 
+pub fn main() {
     // Read the verification keys.
     let vkeys = zkm_zkvm::io::read::<Vec<[u32; 8]>>();
 
     // Read the public values.
     let public_values = zkm_zkvm::io::read::<Vec<Vec<u8>>>();
```

Comment:
What's the public_values here?  https://github.com/GOATNetwork/bitvm2-node/blob/c6f5b0e94356f5b17885b4a951525a58c16b92b5/covenant/aggregation/src/executor.rs#L21?